src/main/game/entities/ICommandable.java

```diff
@@ -11,6 +11,7 @@
     void doTurn();                              // Iterate turn
     Command nextCommand();                      // Next queue for new command or decrement turn count
     Command peekCommand();                      // Peek at next command
+    boolean emptyQueue();                       // Return if queue is empty
```

Comment:
can you change the name to isEmpty() or isQueueEmpty(). My first thought was that this actually emptied the queue.